qiskit/circuit/library/__init__.py

```diff
@@ -196,6 +196,7 @@
    UCRXGate
    UCRYGate
    UCRZGate
+   QFTGate
```

Comment:
I think that also in the API docs it should be with QFT in the basis change section?

Reply:
Thanks! Done